whatshap/cli/haplotag.py

```diff
@@ -57,9 +79,38 @@ def add_arguments(parser):
     arg("--output-haplotag-list", dest="haplotag_list", metavar="HAPLOTAG_LIST", default=None,
         help="Write assignments of read names to haplotypes (tab separated) to given "
         "output file. If filename ends in .gz, then output is gzipped.")
-    arg("--tag-supplementary", default=False, action="store_true",
-        help="Also tag supplementary alignments. Supplementary alignments are assigned to the "
-        "same haplotype as the primary alignment (default: only tag primary alignments).")
+    arg("--tag-supplementary",
+        nargs="?",
+        type=SupplementaryHaplotaggingStrategy,
+        choices=list(SupplementaryHaplotaggingStrategy),
+        default=SupplementaryHaplotaggingStrategy.SKIP, const=SupplementaryHaplotaggingStrategy.COPY_PRIMARY,
+        dest="supplementary_strategy",
+        help="Also tag supplementary alignments. "
+             "Depending on the strategy chosen supplementary alignments are assigned to: "
+             " (i) skip / no flag specified -- no assignment, skipped \n"
+             " (ii) copy-primary / flag without value -- assigned to the same phase block and haplotype, "
+             " as a primary alignment, if any (only for supplementary alignments on the same chromosome as a primary)"
+             " (iii) independent-or-skip -- each supplementary alignment treated as an independent alignment segment "
+             " and is assigned (if possible) to PS/HP based on spanned phased variants. "
+             " If no assignment based on variants is possible, not assignment is performed"
+             " (iv) independent-or-copy-primary -- each supplementary alignment treated as an independent alignment "
+             " segment and is assigned (if possible) to PS/HP based on spanned phased variants. "
+             " If no assignment based on variants if possible, defaults to copying primary alignmtn assignment, "
+             " if exists and on the same chromosome."
+             " [default: with no flag -- skip, default with flag and no value -- copy-primary] ")
```

Comment:
Please use round parentheses for consistency with the other help texts. Also, please remove the space before the `"`.

Reply:
fixed